### ISSUE TITLE
Getting Started tutorial screen 5 Preferences window

### ISSUE THREAD

**Issue by mjross** [Komodo/KomodoEdit#2439]
### Short Summary
The Getting Started tutorial keeps insisting that I open the Preferences window even though I have done so.

### Steps to Reproduce
When working my way through the "Getting Started" tutorial, on screen 5 ("Changing Preferences"), I get the warning message at the bottom "Uh oh / You need to open the Preferences window first", even though I had opened the Preferences window during my previous session within Komodo IDE and going through the tutorial up to that point. So I again opened the Preferences window, looked at some of the sections, and within it pressed the "Cancel" button. At that point I wanted to proceed to screen 6, but without clicking the "Continue Anyway" link, since that pops up a warning message, "Are you sure you want to continue anyway? This tutorial may not work properly if you do not properly follow the tasks given." So I clicked the "Previous" button to go back to screen 4, and then the "Next" button to return to screen 5. But then when I click its "Next" button to go to screen 6, I receive the same message that I need to open the Preferences window first even though I had done so.

### Platform Information
Komodo IDE 10.2.1 on Windows 10 Pro

### Attachments
![getting started tutorial screen 5](https://cloud.githubusercontent.com/assets/829492/24593131/d111cf4e-17e5-11e7-8fc2-ca8c17d26650.png)

**Comment by Naatan** [comment]
Can you reproduce this using `Help > Troubleshooting > Restart in Safe-mode` ?

Please verify that when restarting in safe-mode Komodo should be using the default UI (all your settings would be temporarily reset).

You can start the tutorial again via the Tutorals > Getting Started item in your toolbox (right pane).

**Comment by mjross** [comment]
Yes, I restarted in safe mode, proceeded through the same tutorial again, reached screen 5, viewed the Preferences window, exited that, clicked the Next button, and again received the warning that I had not looked at the Preferences window.

**Comment by Naatan** [comment]
Keep the preferences window open, the tutorial does not instruct you to close it.

**Comment by mjross** [comment]
Okay. It worked. Thank you for the info. I suspect many users are not going to realize that they are supposed to keep the Preferences window open, especially as there is no instruction to do so. As I did, they may look at the many options within the Preferences, perhaps make some changes, and then, satisfied with what they have done, close it and go back to the tutorial. Perhaps that tutorial step could tell the user to keep the Preferences window open and then continue to the next tutorial screen.
